deepspeed/runtime/comm/ccl.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+import numpy as np
+import torch
+# import torch_npu
+import deepspeed.comm as dist
+from deepspeed.accelerator import get_accelerator
+from deepspeed.ops.op_builder.xpu import PackbitsBuilder
```

Comment:
could this import be made accelerator agnostic?  My understanding is any accelerator implement packbitsbuild would be able to support 1bit adam